Automattic/wp-calypso
# Stats: Change support links to open in a new tab

Related to https://github.com/Automattic/red-team/issues/199

## Proposed Changes

* Updating external jetpack.com and wordpress.com support links to open in a new tab

## Why are these changes being made?


* This is a problem because interrupts the ongoing experience and, in some cases, the support link doesn’t allow users to go back using the browser “Back” action.

## Testing Instructions



* verify that the inline support links open in a new tab

<img width="624" alt="Screenshot 2024-11-19 at 11 02 05 PM" src="https://github.com/user-attachments/assets/b35bfaf6-94bc-4499-9737-44669c7a0a41">

## Pre-merge Checklist
